lib/parse-result.js

```diff
@@ -15,6 +15,8 @@ module.exports = function (options) {
       throw new errors.Forbidden('`parseI18nResult` can only be run on a `after` hook.')
     }
 
+    if (!options.language) options.language = 'en'
```

Comment:
I don't think this is needed, as the `options = Object.assign({}, defaults, options)` would assign a default.